wwwroot/config.json

```diff
@@ -24,33 +30,33 @@
 
     // Pop-up disclaimer that will be shown at the start
     // The text of the disclaimer is in lib/Views/GlobalDisclaimer.html
-    // globalDisclaimer: {
-    //     confirmationRequired: false,
-    //     buttonTitle: "I agree",
-    //     title: "Disclaimer",
-    //     // If the current browser location is not prod, or is dev, then show a "not the real site" warning.
-    //     // The text of that is in lib/Views/DevelopmentDisclaimerPreamble.html
-    //     //prodHostRegex: "gov.\\.au$",
-    //     //devHostRegex: "\\b(staging|preview|test|dev)\\.",
-    //     enableOnLocalhost: true // If false, don't show this when running on localhost
-    // },
+    "globalDisclaimer": {
+        "confirmationRequired": false,
+        "buttonTitle": "Go explore",
+        "title": "Welcome",
+        // If the current browser location is not prod, or is dev, then show a "not the real site" warning.
+        // The text of that is in lib/Views/DevelopmentDisclaimerPreamble.html
+        //prodHostRegex: "gov.\\.au$",
+        //devHostRegex: "\\b(staging|preview|test|dev)\\.",
+        "enableOnLocalhost": true // If false, don't show this when running on localhost
+    },
 
     /*
         developerAttribution: {
             text: "Your organisation",
             link: "http://www.example.com"
         },*/
     // This is used in various text messages. You should also set <title> in index.html
-    "appName": "Terria Map",
+    "appName": "eWaterCycle explorer",
 
     // See comments in lib/Views/global.scss for further information about customising colors, logos, fonts etc.
     "brandBarElements": [
       "",
-      "<a target=\"_blank\" href=\"https://terria.io\"><img src=\"images/terria_logo.png\" height=\"52\" title=\"Version: {{version}}\" /></a>",
+      "<a target=\"_blank\" href=\"https:/ewatercyclre.org\"><img src=\"https://github.com/eWaterCycle/ewatercycle/raw/main/docs/examples/logo.png\" height=\"52\" title=\"Version: {{version}}\" /></a>",
       ""
     ],
     // Shown when things go wrong.
-    "supportEmail": "help@example.com",
+    "supportEmail": "help@ewatercycle.org",
```

Comment:
Who owns this eamil address?

Reply:
Dunno, switched to ewatercycle@esciencecenter.nl which lands a a O365 groups email you are a member of. Email is also used on https://www.ewatercycle.org/contact/